_security/configuration/security-installation.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Security installation
```

Comment:
Enabling security

Reply:
See comment below...

Reply:
I think this was regarding the name change, thats updated now